scss/modules/_homepage.scss

```diff
@@ -3,15 +3,61 @@
 // Campaign Finance Data Homepage Styles
 .slab.slab--primary .blue {
```

Comment:
I would suggest to add add a `slab--homepage` class, so then this could be defined as `.slab--primary.slab--homepage`, so then the class can be re-used for specific homepage slab coloring beacuse the link color also changes for the secondary slab on the homepage.